providers/aix/kernel_aix.go

```diff
@@ -49,10 +49,10 @@ func getKernelVersion() (int, int, error) {
 }
 
 // KernelVersion returns the version of AIX kernel
-func KernelVersion() (string, error) {
+func KernelVersion() (int, int, string, error) {
```

Comment:
With this being an exported function we cannot change the signature without introducing a breaking change.

Also, the other providers (linux, darwin, windows) have similar functions with the `KernelVersion() (string, error)` signature. So change this would make AIX inconsistent with them.